pkg/operatorsource/handler.go

```diff
@@ -59,26 +59,47 @@ func (h *operatorsourcehandler) Handle(ctx context.Context, in *v1alpha1.Operato
 		"name":      in.GetName(),
 	})
 
-	reconciler, err := h.factory.GetPhaseReconciler(logger, in)
+	prePhaseReconciler, err := h.factory.GetPrePhaseReconciler(logger, in)
 	if err != nil {
 		return err
 	}
 
-	out, status, err := reconciler.Reconcile(ctx, in)
+	out, status, err := prePhaseReconciler.Reconcile(ctx, in)
```

Comment:
potentially re-declaring of err variable here no?

Reply:
`a short variable declaration may redeclare variables provided they were originally declared earlier in the same block. Redeclaration does not introduce a new variable; it just assigns a new value to the original.`

More about it - https://golang.org/ref/spec#Short_variable_declarations